charts/kyverno/templates/_helpers.tpl

```diff
@@ -77,9 +77,9 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
 {{- end -}}
 
-{{/* Set if a default policy is managed */}}
-{{- define "kyverno.podSecurityDefault" -}}
-{{- if or (eq .Values.podSecurityStandard "default") (eq .Values.podSecurityStandard "restricted") }}
+{{/* Set if a baseline policy is managed */}}
+{{- define "kyverno.podSecurityBaseline" -}}
```

Comment:
@realshuting the `helm-test` failed because I renamed this. Is this something we need to change to `kyverno.podSecurityBaseline` or not?